.changeset/bright-pillows-build.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-explore': patch
+---
+
+Added ability to customize the explore plugin tools tab content
```

Comment:
Need to update this now, just a mentioned that an example was added to the readme